content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -257,6 +257,18 @@ well as lower-trust users.The following listed controls should be enforced/disal
 				undefined / nil<br>
 			</td>
 		</tr>
+		<tr>
+			<td>Capabilities</td>
+			<td>
+				Some capabilities in the <a href="https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities">default set</a> can enable attacks in certain scenarios and must be dropped.<br>
+				<br><b>Restricted Fields:</b><br>
+				spec.containers[*].securityContext.capabilities.drop<br>
+				spec.initContainers[*].securityContext.capabilities.drop<br>
+				<br><b>Allowed Values:</b><br>
+				all<br>
```

Comment:
Note that this does not require updating https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/restricted-psp.yaml because that YAML already specifies requiredDropCapabilities: ALL